### DISCUSSION TITLE
Feature Request: Name discs in multi-disc collections

### DISCUSSION THREAD

**Discussion by NeuroDawg** [navidrome/navidrome#3165] [Ideas]
As the title suggests, I would like to see names of discs in multi-disc collections.

For example, the box set 'Unearthed' by Johnny Cash. It is a multi-disc release, and each disc has it's own name ('Unearthed, Vol. 1: Who's Gonna Cry', 'Unearthed, Vol. 2: Trouble in Mind', 'Unearthed, Vol. 3: Redemption Songs', etc.)

My suggestion would be to read the DISCSUBTITLE tag and use it to give a title to the disc, rather than just showing the disc number as is done currently.

**Comment by deluan** [comment]
This was already implemented a while ago: https://github.com/navidrome/navidrome/issues/272

![Screenshot 2024-07-25 at 4 58 15 PM](https://github.com/user-attachments/assets/cc2d925d-e1f3-4523-a858-b089711fdf81)

**Reply by NeuroDawg** [reply]
Is it using the the DISCSUBTITLE tag? If so, I've to investigate and make sure my tags are correct in my files, because the names weren't picked up for me. I'll get back with results.

Edited: Thank you very much. It helps if I don't misspell the tag when I add it to my files. DISCSUBTITLE is distinctly different than DICSUBTITLE.

Thanks for this feature!
